### DISCUSSION TITLE
New translation English (Great Britain)

### DISCUSSION THREAD

**Discussion by davet2001** [home-assistant/frontend#8185] [Localization]
I would like to contribute an 
English (Great Britain) 
translation

This is ISO 639-1 en-uk 

It is very similar to English (USA), but we spell some words differently e.g. ‘colour’, ‘centre’, ‘aluminium’ and sometimes use different words (wrench=spanner).

Is there a way to have a translation mirror English apart from specific words?

**Comment by TomBrien** [comment]
I'm happy to put my hand up to contribute to this translation, I already do the iOS translation for en-gb. By default strings that are not translated will use the base language (en-us), so the translation is super simple (or just a bunch of Alt+0 when going through the list).

**Reply by davet2001** [reply]
Thanks @TomBrien, I am not sure what the first step is with making this happen.  If anyone can give some pointers that would be greatly appreciated.
